jenkins/toggle-webhook-processor.py

```diff
@@ -0,0 +1,63 @@
+import boto3
+import sys
+import logging
+import click
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+@click.command()
+@click.option(
+    '--enabled',
+    help="Toggle option for the webhook-processor Lambda. "
```

Comment:
nit: should now read something like "Set the state of the trigger for the webhook-processor lambda."